src/ChunkMap.h

```diff
@@ -486,7 +493,12 @@ class cChunkMap
 	cChunkPtr GetChunk(int a_ChunkX, int a_ChunkZ);
 
 	/** Constructs a chunk and queues the chunk for loading if not valid, returning it; doesn't generate */
-	cChunkPtr GetChunkNoGen(int a_ChunkX, int a_ChunkZ);
+	// Deprecated in favor of the vector version
+	cChunkPtr GetChunkNoGen(int a_ChunkX, int a_ChunkZ)
+	{
+		return GetChunkNoGen(cChunkCoords(a_ChunkX, a_ChunkZ));
+	}
+	cChunkPtr GetChunkNoGen(cChunkCoords a_Chunk);
```

Comment:
I think the `cChunkCoords` version should go directly under the doxy comment, and the deprecated version can go without an explicit doxy-comment.